resources/views/layouts/_nav.blade.php

```diff
@@ -5,7 +5,7 @@
                 <div class="block lg:flex lg:items-center lg:shrink-0">
                     <div class="flex justify-between items-center p-4 lg:p-0">
                         <a href="{{ route('home') }}" class="mr-4">
-                            <img class="h-6 w-auto lg:h-8" src="{{ asset('images/laravelio-logo.svg') }}" alt="" />
+                            <img loading="lazy" class="h-6 w-auto lg:h-8" src="{{ asset('images/laravelio-logo.svg') }}" width="330" height="78" alt="{{ config('app.name') }}" />
```

Comment:
We can't use width/height here because it's already set through Tailwind classes.

Reply:
I'd also prefer not to lazy load the logo here.

Reply:
The width and height attributes are overridden by Tailwind. But at least, the layout won't shift when the image loads. That's why Google asks for this (as I said though, they test on slow hardware and network).